### PR TITLE
Add benefits to README

### PR DESCRIPTION
**Pull request by kaspth**
Still WIP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,41 @@ Oaken is fixtures + factories + seeds for your Rails development & test environm
...
+- You can name things: don't lose hours debugging factory-based tests because everything's anonymous.
+- You can show model connections more easily:
+- Optimizing goes into the foundation, not a local maxima. When you optimize
+- Better Discoverability:
+
+### Benefits over fixtures
+
+- Still fast tests! Oaken inserts data before tests run & wraps tests in transactions, just like fixtures.
+- No more YAML: use Ruby to lay out your object graph sequentially with better organization.
+- Group by Scenario: split data into semantic files so there's no more data mixing within a file anymore.
```
Comment (tcannonfodder) [inline README.md]: I'm not exactly sure what you mean here. But going to try to take a stab!

```suggestion
- Break out edge cases or complex scenarios: split data for edge cases or tricky test setups into their own cases, so you don't end up with a 500-line fixture file peppered with warts.
```

File: README.md [hunk 2]
```diff
@@ -4,6 +4,41 @@ Oaken is fixtures + factories + seeds for your Rails development & test environm
...
+  - Tests mirror what developers have already seen in their dev browser.
+  - Easier developer onboarding by building a proper mental model faster.
+  - Less code that's easier to maintain with more visibility into your object graph.
+
+### Benefits over factories
+
+- Fast tests! Seed shared records once and reuse them across tests, transactions rollback changes.
+- Sets you up for the long term: factories drain your mental resources quicker 
+- Visibility into your system: factories optimize for isolation, which make it easy to get started, but becomes more complex over time.
+- Less mental load: Schrödinger's Object definition
```
Comment (tcannonfodder) [inline README.md]: I like the term, but it feels like its needs concrete. examples to drive the point home

File: README.md [hunk 3]
```diff
@@ -4,6 +4,41 @@ Oaken is fixtures + factories + seeds for your Rails development & test environm
...
+- In tests, reuse development seed data:
+  - Tests mirror what developers have already seen in their dev browser.
+  - Easier developer onboarding by building a proper mental model faster.
+  - Less code that's easier to maintain with more visibility into your object graph.
+
+### Benefits over factories
+
+- Fast tests! Seed shared records once and reuse them across tests, transactions rollback changes.
+- Sets you up for the long term: factories drain your mental resources quicker 
+- Visibility into your system: factories optimize for isolation, which make it easy to get started, but becomes more complex over time.
```
Comment (tcannonfodder) [inline README.md]: Also worth noting that you DRY up your tests _more_ for non-trivial systems because you write the Oaken seed for common cases _once_, rather than per-test/file

**Comment by mikker** [comment]
Congrats on 1.0! 🎉

**Comment by kaspth** [comment]
Thank you!
